### ISSUE TITLE
Hey Kirill!

### ISSUE THREAD

**Issue by resistorsoftware** [kirillplatonov/shopify-hotwire-sample#15]
Love your codez! So nice to see contributors that are current, filling in for the inactive ghosts of Shopify past.

I have a basic question for you. In production you set a config force_iframe true, but in development and test environments it is configurable and false. 

What is the point of the force_iframe? I am not sure I follow the pattern there. I get it that embedded Apps use the iframe element, and nothing changes there. Is this a custom flag for people using non-embedded Apps outside any iframe?

**Comment by kirillplatonov** [comment]
Hey! Thanks, man.

This flag is used to optionally skip redirect to Shopify Admin iframe and to skip JWT stuff. The main purpose is to be able to write controller & system tests for embedded app. It can also be used in development for example when you're working offline (I used it only once on the plane 🙂). 

I explained the approach that I use for testing embedded apps in my blog: https://kirillplatonov.com/posts/testing-shopify-apps-in-rails/
